README.md

```diff
@@ -369,6 +370,56 @@ Properties expanded by Jenkins:
 
 If you want to learn more about ACR tasks, [here is the documentation](https://docs.microsoft.com/en-gb/azure/container-registry/container-registry-tasks-reference-yaml).
 
+## Contract testing with Pact
+
+#### Usage
+
+You can activate contract testing lifecycle hooks in the CI using the `enablePactAs()` method.
+
+The different hooks are based on roles that you can assign to your project: `CONSUMER` and/or `PROVIDER`. A common broker will be used as well as the naming and tagging conventions.
+
+Here is an example of a project which acts a consumer and provider (for example a backend-for-frontend):
+
+```groovy
+import uk.gov.hmcts.contino.AppPipelineDsl.PactRoles as PactRoles
+
+/* … */
+
+withInfraPipeline(product) {
```

Comment:
why would you be doing pact testing in your infra pipeline?
```suggestion
withPipeline(product) {
```

Reply:
Woops, thx